### ISSUE TITLE
Ionic 4 + Socket.io on Heroku

### ISSUE THREAD

**Issue by manel00** [rodgc/ngx-socket-io#41]
Hi,

First of all thanks for the plugin, but i'm facing this problem and i'm doing whatever to fix it but cannot get any solution, what i'm doing wrong?

> Access to XMLHttpRequest at 'https://server.herokuapp.com/socket.io/?EIO=3&transport=polling&t=Mnn5eIt' from origin 'http://localhost:8100' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.

Thank you so much

**Comment by webcat12345** [comment]
Hello @manel00 
I have the same issue.
Did you find a solution?

**Comment by ctfrancia** [comment]
if this issue, hasn't been resolved then please open up a new ticket, as the last response is 5 months ago

**Comment by LeonardoRick** [comment]
Maybe you should try to enable cors on your server side

```
import express from 'express';
import cors from 'cors';
const app = express();
app.use(cors());
```
